### ISSUE TITLE
`links` command allow bulk view

### ISSUE THREAD

**Issue by pete-ts** [unisonweb/unison#1397]
In a namespace where a number of entities have been added, and then  author/license/docs have been linked, it would be handy if `links` could allow bulk mode, so that the developer can double check if all values have been linked correctly:

```
.root.prs.charGenerators.test.v1.gen.Char> ls

  1.  alpha              ('{Gen} Char)
  2.  alpha/             (2 definitions)
  3.  ascii              ('{Gen} Char)
  4.  ascii/             (2 definitions)
  5.  asciiNonPrintable  ('{Gen} Char)
  6.  asciiNonPrintable/ (2 definitions)
  7.  asciiPrintable     ('{Gen} Char)
  8.  asciiPrintable/    (2 definitions)
  9.  digit              ('{Gen} Char)
  10. digit/             (2 definitions)
  11. hexDigit           ('{Gen} Char)
  12. hexDigit/          (2 definitions)
  13. lower              ('{Gen} Char)
  14. lower/             (2 definitions)
  15. patch              (patch)
  16. upper              ('{Gen} Char)
  17. upper/             (2 definitions)

.root.prs.charGenerators.test.v1.gen.Char> links 3

  1. .base.metadata.authors.pete_ts           : Author
  2. .base.metadata.licenses.pete_ts.mit_2020 : License
  3. ascii.doc                                : Doc
  
  Tip: Try using `display 1` to display the first result or `view 1` to view its
       source.

```
But at the same time:

```
.root.prs.charGenerators.test.v1.gen.Char> links 1 3

  ❓
  
  I couldn't resolve any of these symbols:
  
      1 | .root.prs.charGenerators.test.v1.gen.Char.ascii
  
  Using these fully qualified names:
    .root.prs.charGenerators.test.v1.gen.Char.ascii
```

**Comment by aryairani** [comment]
First step would be to design the output.  What exactly should we see?

**Comment by pete-ts** [comment]
My thinking is that this should be a cross between the current behaviour of "updating a link" which prints something along the lines of:
```
  Updates:
  
    1. .my.namespace.x.y.z : a -> Boolean
       + 2. x.doc : Doc
```
and the single `links X` current behaviour. Currently, in order to be able to check if multiple functions have the correct links the user has to `ls` -> `links 1` -> `ls` -> `links 3` -> `ls` -> `...`

(An example namespace that I happen to have at hand atm)
```
.root.prs.charGenerators.test.v1.gen.Char> ls

  1.  alpha              ('{Gen} Char)
  2.  alpha/             (2 definitions)
  3.  ascii              ('{Gen} Char)
  4.  ascii/             (2 definitions)
  ... truncated...
```
The desired output could be something along the lines of:
```
  1. alpha: 
     -> 2. .base.metadata.authors.pete_ts           : Author
     -> 3. .base.metadata.licenses.pete_ts.mit_2020 : License
     -> 4. alpha.doc                                     : Doc
  5. ascii: 
     -> 6. .base.metadata.authors.pete_ts           : Author
     -> 7. .base.metadata.licenses.pete_ts.mit_2020 : License
     -> 8. ascii.doc                                     : Doc
  9. some unlinked function:
     -> (not sure here)
```

Overview of `show me all the links in the namespace` would allow the following use cases (command section below):
* `edit/view 1`       -- I can view and edit my function
* `view/display 3`  -- is this the right content?
* `display 4`          -- I need to read the doc to check for typos
* `unlink 4 1`         -- I have typos in my doc so I unlink to update
* `link 6-7 9`             -- I was indeed missing a link and now it is ok

(I am aware that `unlink 4 1` does not work that way atm and the same for `link 6-7 9`)

**Comment by mitchellwrosen** [comment]
fixed in #4574
